ipapython/dnsutil.py

```diff
@@ -447,6 +451,8 @@ def check_zone_overlap(zone, raise_on_error=True):
     except dns.exception.DNSException as e:
         msg = ("DNS check for domain %s failed: %s." % (zone, e))
         if raise_on_error:
+            if isinstance(e, dns.resolver.NoNameservers):
+                raise DNSNoNameservers(**e.args, **e.kwargs)
```

Comment:
``**args`` looks wrong. You can use ``from None`` to suppress exception chaining (another error occurred while handling ...).

```suggestion
                raise DNSNoNameservers(*e.args, **e.kwargs) from None
```

Reply:
Thanks, fixed. This was really an idea to try but I am still to reproduce the original issue myself.